bentoml/bentos.py

```diff
@@ -192,7 +193,7 @@ def predict(request_data: np.ndarray):
     build_ctx = os.getcwd() if build_ctx is None else os.path.realpath(build_ctx)
     svc = load(svc_import_str, working_dir=build_ctx)
 
-    models = [] if models is None else models
+    additional_models = [] if additional_models is None else additional_models
```

Comment:
```suggestion
    additional_models = additional_models or []
```

Reply:
Not a huge fan of hacking `or` like this, even if it would be safe.